### PR TITLE
ci: release workflow support v0.9

### PR DESCRIPTION
**Pull request by Yaiba**
This pr changes release workflow to support kwil-db v0.9

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -21,8 +21,8 @@ jobs:
         shell: bash
         run: |
           version=`echo ${{ github.event.release.tag_name }} | sed 's/^v//' | cut -d '.' -f 2`
-          kgw_version=$(v8="3"; v7="2"; v6="1"; eval echo \${v$version})
-          kjs_version=$(v8="7"; v7="6"; v6="5"; eval echo \${v$version})
+          kgw_version=$(v9="3"; v8="3"; v7="2"; v6="1"; eval echo \${v$version})
+          kjs_version=$(v9="8"; v8="7"; v7="6"; v6="5"; eval echo \${v$version})
```
Comment (Yaiba) [inline .github/workflows/release.yaml:25]: @KwilLuke Just to confirm, kwil-db v0.9 will need kwil-js v0.8 ?

Comment (Yaiba) [reply .github/workflows/release.yaml:25]: cool

**Comment by Yaiba** [comment]
https://github.com/kwilteam/kwil-db/actions/runs/11000884839 test v0.9 with kgw `main`(v0.3.3) and kwil-js `main`,  passed.

kgw v0.3.3 is compatible with kwil v0.9, and kwil-js is [forward compatible with kwil v0.9](https://github.com/kwilteam/kwil-db/pull/1011#discussion_r1773683491)

**Review by jchappelow — APPROVED** [review]
LGTM if Luke confirmed.

**Comment by KwilLuke** [comment]
Sorry, just to clarify @Yaiba @jchappelow -

Kwil-js v0.9 is not released or tagged. Should this ci use kwil-js v0.8?

This would mean that this change should be reverted:

```
- kjs_version=$(v8="7"; v7="6"; v6="5"; eval echo \${v$version})
+ kjs_version=$(v9="8"; v8="7"; v7="6"; v6="5"; eval echo \${v$version})
```

**Comment by Yaiba** [comment]
> Sorry, just to clarify @Yaiba @jchappelow -
> 
> Kwil-js v0.9 is not released or tagged. Should this ci use kwil-js v0.8?
> 
> This would mean that this change should be reverted:
> 
> ```
> - kjs_version=$(v8="7"; v7="6"; v6="5"; eval echo \${v$version})
> + kjs_version=$(v9="8"; v8="7"; v7="6"; v6="5"; eval echo \${v$version})
> ```

It's fine, this workflow is only used if we **release**(not just tag) a v0.9 version in kwil-db,

**Comment by KwilLuke** [comment]
Okay - I think the kwil-db release will likely happen before the kwil-js release. Authenticated RPC is close, but we prob won't be ready to release until end of this week or early next.

**Comment by Yaiba** [comment]
> Okay - I think the kwil-db release will likely happen before the kwil-js release. Authenticated RPC is close, but we prob won't be ready to release until end of this week or early next.

hmm, still ndb, this won't affect our release process, this workflow will just fail if kwil-js is released after kwil-db.

**Pull request merged**